public/js/index.js

```diff
@@ -1458,29 +1458,26 @@ $(document).ready(function () {
     });
 
     // Helpers.
-    $('.delete-button').click(function () {
-        var $this = $(this);
-        var filter = "";
-        if ($this.attr("id")) {
-          filter += "#"+$this.attr("id")
+    $('.delete-button').click(showDeletePopup);
+
+    var prev_val = "{{branch_name}}";
```

Comment:
Also it would be better to use ${name} (ES6 template syntax) to be less confusing with golang templates